### PR TITLE
Fix for banner and too far scrolling

### PR DESCRIPTION
**Pull request by GilgusMaximus**
---
Fix for banner and too far scrolling
---

**Pull Request Type**
Please select what type of pull request this is:
- [x] Bugfix
- [ ] Feature Implementation

**Related issue**
From dev channel

**Description**
The last fix for the overflowing banner introduced a bug that lets you scroll past the video section on any page.
This is now fixed by a small restructuring of how the app layout works and how it is styled

**Testing (for code that is not small enough to be easily understandable)**
Tested with and without banner

**Desktop (please complete the following information):**
 - OS: Win
 - OS Version: 10
 - FreeTube version: v0.17.0-RC

### CODE REVIEW

File: src/renderer/App.css [hunk 1]
```diff
@@ -44,11 +44,12 @@
...
+    width: 80%;
+    margin-top: 60px;
   }
 
   .flexBox {
     margin-top: 60px;
     margin-bottom: -75px;
   }
-}
+}
```
Comment (PikachuEXE) [inline src/renderer/App.css:55]: I think we do have `.editorconfig` to ensure there is a final newline char...
Maybe your editor/IDE need to be setup to support it
https://github.com/FreeTubeApp/FreeTube/blob/development/.editorconfig#L9
https://editorconfig.org/#download

**Review by PikachuEXE — APPROVED** [review]
Test locally (for this branch no the one for dev branch)

**Review by PikachuEXE — CHANGES_REQUESTED** [review]
I forgot my line comment...

File: src/renderer/App.css (hunk 1 above)
Comment (GilgusMaximus) [reply src/renderer/App.css]: Updated.

**Review by PikachuEXE — APPROVED** [review]

**Review by ChunkyProgrammer — APPROVED** [review]
Lgtm

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — APPROVED** [review]

**Pull request merged**